README.md

```diff
@@ -918,16 +922,16 @@ See our [contribution documentation](https://doc.nuxeo.com/x/VIZH).
 * [Node.js](http://nodejs.org/#download)
 * [gulp](http://gulpjs.com/)
 * [Bower](http://bower.io/)
-* [npm](https://www.npmjs.com/)
+* [yarn](https://yarnpkg.com/)
 
 ### Setup
 
-Install [Node.js](http://nodejs.org/#download) and then use `npm` to install all the required
+Install [Node.js](http://nodejs.org/#download) and then use `yarn` to install all the required
```

Comment:
Not `npm` or `yarn` here?

Reply:
No, that's for contributing, so `yarn` is needed.